backend/de.metas.business/src/test/java/de/metas/document/dimension/InvoiceLineDimensionFactoryTest.java

```diff
@@ -29,6 +31,8 @@ void get_and_set()
 		final Dimension dimensionFromRecordExpected = dimension.toBuilder()
 				.userElement1Id(0)
 				.userElement2Id(0)
+				.userElementNumber1(BigDecimal.ZERO)
```

Comment:
nop! those shall be null!

Reply:
fix ur de.metas.document.dimension.InvoiceLineDimensionFactory#getFromRecord implementation